site/en/docs/extensions/mv3/getstarted/development-basics/index.md

```diff
@@ -0,0 +1,271 @@
+on the refresh icon next to the **on/off** toggle:
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/QNtwZICbwsGO3gacCJEd.png", 
+alt="Reload an extension", width="500", height="233", class="screenshot" %}
+
+{% Aside %}
+
+💡 **TIP**: Remember to refresh your extension every time you make changes.
+
+{% endAside %}
+
+## Finding console logs and errors {: #logs }
+
+### Console logs {: #console}
+
+During development, you can debug your code by accessing the browser console logs. In this case, we
+will locate the logs for the popup. Start by adding a script tag to `hello.html`.
+
+{% Label %}hello.html{% endLabel %}
+
+```html
+<html>
+ <head>
+   </head>
+ <body>
+   <h1>Hello Extensions of the world! 👋/h1>
+   <script src="popup.js"></script>
+ </body>
+</html>
+```
+
+Create a `popup.js` file and add the following code to output the log to the console:
+
+{% Label %}popup.js{% endLabel %}
+
+```js
+console.log("This is a popup!")
+```
+
+To see this message logged in the Console, complete the following steps:
+
+  1. Refresh the extension
+  2. Open the popup
+  3. Right-click on the popup
+  4. Select **Inspect**. 
+      <figure> 
+      {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/vHGHW1o4J0kZgUkAteRQ.png", 
+      alt="Inspecting the popup", width="322", height="262", class="screenshot" %}
+        <figcaption>
+        Inspecting a popup 
+        </figcaption>
+      </figure>
+  5. In the [DevTools][dev-tools], navigate to the **Console** panel.
+    <figure>
+    {% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/1ZrcBEYcbMxW1c9UvBy9.png", 
+    alt="DevTools Code Panel", width="400", height="374", class="screenshot" %}
+      <figcaption>
+      DevTools Console Panel 
+      </figcaption>
+    </figure>
+
+### Error logs {: #errors }
+
+Now let's break the extension. We can do so by removing the closing quote in the `popup.js` file:
+
+{% Label %}popup.js{% endLabel %}
+
+```js
+console.log("This is a popup!) // broken code
+```
+
+Go to the Extension Management page and open the popup. An **Errors** button will appear.
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/a9lAHCJZZrebOSKrkPRD.png", 
+alt="Extension Management page with error button", width="300", height="226", class="screenshot" %}
+
+Click on the **Errors** button to learn more about the error:
+
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/GrorLyaC6nRAyqc1qULC.png", 
+alt="Extension error details", width="400", height="281", class="screenshot" %}
+
+To learn more about debugging the service worker, options page, and content scripts, see [Debugging
+extensions][doc-debug].
+
+## Structuring an extension project {: #structure }
+
+There are many ways to structure an extension project; however, you should always place the
+manifest.json file at the **root**. The following is a structure example:
+
+
+```text
+└── my extension/
+    ├── manifest.json
+    ├── background.js
+    ├── scripts/
+    │   └── content.js
+    │   └── jquery-min.js
```

Comment:
jQuery feels a bit dated. Might be best to replace it with something like React.js.

```suggestion
    │   └── react.production.min.js
```